Makefile

```diff
@@ -27,12 +28,12 @@ run_docker:
 
 run_on_specific_files_flag:
 	docker build -t html-parser .
-	docker run -it html-parser python -m cli.run_parser s3://cpr-dev-data-pipeline-cache/marks/ingest_output/ s3://cpr-dev-data-pipeline-cache/marks/parser_output/ --s3 --files "1331.0.json"
+	docker run -it -e AWS_ACCESS_KEY_ID="${AWS_ACCESS_KEY_ID}" -e AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}" html-parser python -m cli.run_parser "${PARSER_INPUT_PREFIX}" "${EMBEDDINGS_INPUT_PREFIX}" --s3 --files "${FILES_TO_PARSE_FLAG}"
```

Comment:
In a future improvement you may wish to add a test to ensure all env vars are set